lib/src/converter.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:fuzzy/fuzzy.dart';
+import 'package:human_duration_parser/human_duration_parser.dart';
+import 'package:nyxx/nyxx.dart';
+import 'package:nyxx_commands/nyxx_commands.dart';
+import 'package:running_on_dart/src/models/docs.dart';
+import 'package:running_on_dart/src/models/feature_settings.dart';
+import 'package:running_on_dart/src/models/reminder.dart';
+import 'package:running_on_dart/src/modules/docs.dart';
+import 'package:running_on_dart/src/modules/reminder.dart';
+import 'package:running_on_dart/src/modules/tag.dart';
+
+import 'models/tag.dart';
+
+final reminderConverter = Converter<Reminder>(
+  (view, context) =>
+      ReminderModule.instance.search(context.user.id, view.getQuotedWord()).cast<Reminder?>().followedBy([null]).first,
```

Comment:
`.cast<Reminder?>().followedBy([null]).first` can now be replaced with `.firstOrNull`